tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -0,0 +1,172 @@
+    expect(wrapper.find(btnAddPerson).exists()).toBeFalsy()
+    expect(wrapper.find(btnAddCompletingParty).exists()).toBeFalsy()
+    expect(wrapper.find(btnAddCorp).exists()).toBeFalsy()
+    wrapper.destroy()
+  })
+
+  it('does not Start by Adding Completing Party Button when people list is not empty', () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList()
+    const wrapper = wrapperFactory()
+    expect(wrapper.find(btnStartAddCompletingParty).exists()).toBeFalsy()
+    wrapper.destroy()
+    resetStore()
+  })
+
+  it('shows Add Person and Add Corporation Button when people list is not empty', () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList()
+    const wrapper = wrapperFactory()
+    expect(wrapper.find(btnAddCorp).exists()).toBeTruthy()
+    expect(wrapper.find(btnAddPerson).exists()).toBeTruthy()
+    wrapper.destroy()
+    resetStore()
+  })
+
+  it('shows Add Completing Party Button when people list is not empty and has no Completing Party', () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList(['Director'])
+    const wrapper = wrapperFactory()
+    expect(wrapper.find(btnAddCompletingParty).exists()).toBeTruthy()
+    wrapper.destroy()
+    resetStore()
+  })
+
+  it('Does not show Add Completing Party Button when people list is not empty and has Completing Party', () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList()
+    const wrapper = wrapperFactory()
+    expect(wrapper.find(btnAddCompletingParty).exists()).toBeFalsy()
+    wrapper.destroy()
+    resetStore()
+  })
+
+  it('Sets the data attributes as expected when add button is clicked', async () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList()
+    const wrapper = wrapperFactory()
+    wrapper.find(btnAddPerson).trigger('click')
+    await wrapper.vm.$nextTick()
+    expect(wrapper.vm.$data.showOrgPersonForm).toBe(true)
+    expect(wrapper.vm.$data.addEditInProgress).toBe(true)
+    expect(wrapper.vm.$data.nextId).toBe(1)
+    wrapper.destroy()
+    resetStore()
+  })
+
+  it('Shows the add person form when add person button is clicked', async () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList()
+    const wrapper = wrapperFactory()
+    wrapper.find(btnAddPerson).trigger('click')
+    await wrapper.vm.$nextTick()
+    expect(wrapper.find(appointForm).exists()).toBeTruthy()
+    wrapper.destroy()
+    resetStore()
+  })
+
+  it('Shows check mark next to roles added', () => {
+    store.state.stateModel.addPeopleAndRoleStep.orgPeople = getPersonList(['Director', 'Incorporator'])
+    const wrapper = wrapperFactory()
+    expect(wrapper.find(checkIncorporator).exists()).toBeTruthy()
+    expect(wrapper.find(checkDirector).exists()).toBeTruthy()
+    expect(wrapper.find(checkCompletingParty).exists()).toBeFalsy()
+    wrapper.destroy()
+    resetStore()
+  })
+})
```

Comment:
Should also have tests for Show Errors.

Reply:
Should also have tests for role logic (see role methods) and the other methods.

Reply:
As per discussions, we will expand on the unit testing in the next pr! 👍